tests/test_base.py

```diff
@@ -34,6 +35,10 @@ def test_passwd_present(auto_container):
     assert auto_container.connection.file("/etc/passwd").exists
 
 
+@pytest.mark.skipif(
+    not PODMAN_SELECTED,
+    reason="inconsistent image size reporting between docker and podman",
```

Comment:
maybe something like `docker size reporting is dependant on underlying filesystem`